packages/react-reconciler/src/__tests__/ReactHooks-test.internal.js

```diff
@@ -27,6 +27,7 @@ describe('ReactHooks', () => {
 
     ReactFeatureFlags = require('shared/ReactFeatureFlags');
     ReactFeatureFlags.debugRenderPhaseSideEffectsForStrictMode = false;
+    ReactFeatureFlags.enableStableConcurrentModeAPIs = true;
```

Comment:
If you rebase on master, you can wrap the concurrent tests with an `__EXPERIMENTAL__` check instead.